packages/bpk-component-input/src/BpkClearButton-test.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import BpkClearButton from './BpkClearButton';
+
+describe('BpkClearButton', () => {
+  it('should render correctly', () => {
+    const tree = renderer
+      .create(<BpkClearButton label="Clear" onClick={() => null} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
Copied from `BpkCloseButton` with minimal changes.

Reply:
could you add a test to add a custom class? this will bring code coverage to 100% for this one